backend/app/GraphQL/Mutations/Login.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\GraphQL\Mutations;
+
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+
+class Login
+{
+    /**
+     * @param  null  $_
+     * @param  array<string, mixed>  $args
+     */
+    public function __invoke($_, array $args): User
+    {
+        // Plain Laravel: Auth::guard()
+        // Laravel Sanctum: Auth::guard(config('sanctum.guard', 'web'))
+        $guard = Auth::guard(config('sanctum.guard', 'web'));
+
+        if ( ! $guard->attempt($args)) {
+            throw new \Error('Invalid credentials.');
```

Comment:
We should probably reference https://lighthouse-php.com/3.1/guides/error-handling.html#additional-error-information to produce a useful graphql error message if the login is unsuccessful.  Right now we just return a 500 Internal Error

Reply:
After discussing, we will be addressing this in a future iteration under a new issue to be created.